rust/helm/scraper/templates/external-secret.yaml

```diff
@@ -23,30 +23,16 @@ spec:
         HYP_BASE_DB: {{ print "'{{ .db | toString }}'" }}
         {{/* RPC URLs */}}
 {{- /*
-   * For each Outbox network, create an environment variable with the RPC endpoint.
+   * For each chain, create an environment variable with the RPC endpoint.
    * The templating of external-secrets will use the data section below to know how
    * to replace the correct value in the created secret.
    */}}
-        {{- range .Values.hyperlane.outboxChains }}
+        {{- range .Values.hyperlane.chains }}
         {{- if not .disabled }}
         {{- if eq .connection.type "httpQuorum" }}
-        HYP_SCRAPER_{{ .name | upper }}_OUTBOX_CONNECTION_URLS: {{ printf "'{{ .%s_rpcs | fromJson | join \",\" }}'" .name }}
+        HYP_BASE_CHAINS_{{ .name | upper }}_CONNECTION_URLS: {{ printf "'{{ .%s_rpcs | fromJson | join \",\" }}'" .name }}
```

Comment:
Not necessary, but now that the scraper doesn't require specific configuration and these env vars are the same as the ones used by the relayers / validators, there's probably an opportunity to bring things like this into the `agent-common` templates and share it

Reply:
Definitely! I'll file a ticket

Reply:
Filed https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1344